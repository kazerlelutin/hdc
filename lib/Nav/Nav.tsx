import classes from "./Nav.module.css";

export default function Nav(){

    return <p>
        sq
    </p>
}