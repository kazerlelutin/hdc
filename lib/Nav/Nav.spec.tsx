import { render, cleanup } from '@testing-library/react';
import Nav from './Nav';

describe('<Nav />', () => {

    afterEach(cleanup);

    it('Check snapshot', () => {
        const { asFragment } = render(<Nav/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
