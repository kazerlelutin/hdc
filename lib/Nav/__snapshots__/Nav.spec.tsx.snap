// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> Check snapshot 1`] = `
<DocumentFragment>
  <p>
    sq
  </p>
</DocumentFragment>
`;
